packages/web/src/pages/chat-page/components/ChatMessagePlaylist.tsx

```diff
@@ -44,9 +44,9 @@ export const ChatMessagePlaylist = ({
   const { data: playlist, status } = useGetPlaylistById(
     {
       playlistId,
-      currentUserId
+      currentUserId: currentUserId!
     },
-    { disabled: !playlistId }
+    { disabled: !playlistId || !currentUserId }
```

Comment:
Not in love with this pattern. Idk if it's possible to make the type of args understand that it's only called after the disabled condition is false. It would be great to have the behavior like

```
const disabled = !playlistId || !currentUserId
if (!disabled) {
	fetch({playlistId, currentUserId})
}
```
in which it would know that by the time we call fetch, the args must exist